chat.schildi.desktop.appdata.xml

```diff
@@ -0,0 +1,62 @@
+  <name>SchildiChat</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>SchildiChat</developer_name>
+  <summary>A Matrix Client based on Element with a more traditional instant messaging experience</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://schildi.chat/</url>
+  <description>
+    <p>SchildiChat-Desktop is a fork of Element.</p>
+    <p>The most important changes of SchildiChat-Desktop compared to Element are:</p>
+    <ul>
+      <li>A unifed chat list for both direct and group chats</li>
+      <li>Message bubbles</li>
+      <li>Bigger items in the room list</li>
+      <li>… and more!</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot>
+      <caption>SchildiChat logged-in frontend</caption>
+      <image type="source">https://raw.githubusercontent.com/SchildiChat/schildichat-desktop/sc/screenshots/1.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.7.14" date="2020-11-30"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
```

Comment:
```suggestion
  <content_rating type="oars-1.1">
    <content_attribute id="social-chat">intense</content_attribute>
    <content_attribute id="social-audio">intense</content_attribute>
    <content_attribute id="social-contacts">intense</content_attribute>
  </content_rating>
```

Reply:
How should/can that be different from upstream?

Reply:
`none` fields can be simply skipped.